### PR TITLE
feat: bump avsc and i18n example files

### PR DESCRIPTION
**Pull request by JimMoen**

### CODE REVIEW

**Comment by JimMoen** [comment]
CI failed temporarily, re-run failed check after `emqx/emqx:v5.7.0` tagged

**Review by zmstone — CHANGES_REQUESTED** [review]
Missing information:

1. Plugin config format (AVRO JSON)
2. Plugin config file initial location
3. Plugin run-time update REST API (maybe provide a `curl` example).
4. UI aid is only available in Enterprise ediition.

**Comment by zmstone** [comment]
Add some get_config example.
Also describe config life-cycle, when is config loaded, before plugin app start/2 callback or after etc.

File: rebar_template.config [hunk 1]
```diff
@@ -12,7 +12,7 @@
     ]}.
 
 {plugins, [
-    {emqx_plugrel, {git, "https://github.com/emqx/emqx_plugrel.git", {tag, "0.3.0"}}}
+    {emqx_plugrel, {git, "https://github.com/emqx/emqx_plugrel.git", {tag, "0.4.0"}}}
```
Comment (zmstone) [inline rebar_template.config:15]:
```suggestion
    {emqx_plugrel, {git, "https://github.com/emqx/emqx_plugrel.git", {tag, "0.4.1"}}}
```

File: priv/config_schmea.avsc.example [hunk 2]
```diff
@@ -0,0 +1,141 @@
+{
```
Comment (zhongwencool) [inline priv/config_schmea.avsc.example:1]: This file is `config_schema.avsc.example` not `schmea` ?

**Comment by zmstone** [comment]
the default config should continue to be hocon.
Remember: hocon is super-set of JSON.

EMQX should type-check the config with avro schema if the schema exists. Otherwise just load it as-is

**Review by HJianBo — APPROVED** [review]

**Review by HJianBo — APPROVED** [review]

**Review by zmstone — APPROVED** [review]

File: emqx-plugin.template [hunk 3]
```diff
@@ -3,20 +3,29 @@
     {description, "Another amazing EMQX plugin."},
     {version, "1.0.0", "The release version of this plugin."},
     {app_vsn, "0.1.0", "The erlang application vsn value."},
-    {emqx_vsn, "v5.4.1", "EMQX version to use as a dependency."},
+    {emqx_vsn, "v5.6.1", "EMQX version to use as a dependency."},
```
[the remark this answers is not in the document]
Comment (JimMoen) [reply emqx-plugin.template:6]: Will be updated when v5.7.0 released

**Pull request merged**